### PR TITLE
Abacus rebase

### PR DESCRIPTION
**Pull request by okeefm**

### CODE REVIEW

**Comment by tstrass** [comment]
I've looked at a couple files like FhirStu3.Java and FhirR4.Java, and I've tested this successfully with FHIR Patient Generator, but since this is too much to really review all of it, are there any other specific diffs you think could affect our stuff, or are there any other places you had to resolve conflicts?

**Comment by okeefm** [comment]
I think those two files are the _big_ touchpoints into Synthea that we've had to change. There were also some merge conflicts I had to resolve in `synthea.properties`, so that's also worth taking a look at. Other than that, if you don't see any major issues, and the rebased version works for you, I think that's probably a decent review?

File: src/main/resources/synthea.properties [hunk 1]
```diff
@@ -5,29 +5,34 @@ exporter.use_uuid_filenames = false
 exporter.subfolders_by_id_substring = false
 # number of years of history to keep in exported records, anything older than this may be filtered out
 # set years_of_history = 0 to skip filtering altogether and keep the entire history
-exporter.years_of_history = 10
+exporter.years_of_history = 2
```
Comment (tstrass) [inline src/main/resources/synthea.properties]: Do we need this to be 10 years for certain measures or is 2 ok?

Comment (okeefm) [reply src/main/resources/synthea.properties]: I believe 2 is ok. That's what it is in the current `abacus` head branch: https://github.com/projecttacoma/synthea/blob/abacus/src/main/resources/synthea.properties#L8

**Review by hossenlopp — APPROVED** [review]
Good for force push!

**Review by tstrass — APPROVED** [review]

**Pull request merged**